20.md

```diff
@@ -8,22 +8,42 @@ Web Comments
 
 A special event with kind `34` is defined, meaning "web comment".
 
-Events of this kind should have at least one `r` tag with the value set to the normalized URL of the webpage they're commenting in.
+Events of this kind should have both or at least one of the following tags:
 
-The content should be the plaintext of the comment.
+1- a `r` tag with the value that represents some kind of identifier for the object being commented on. 
+No specific format is enforced, but it preferably should be something that won't change later.  
+A possible format could be: `"<MY WEBSITE NAME> <POST TYPE> <POST ID>"`  
+So when used it'll look something like: `"Github Pull_Request 19"`.  
+This identifier can be placed in a `<meta name='identifier' content='...' />` tag on its page.  
+This will help later if someone wants to create a nostr commenting extension or widget that automatically injects comments on each page.  
+So this way this widget will be able to detect which filter it should be using in the query.  
 
-For example:
+2- a `u` tag with the value set to the normalized URL of the webpage they're commenting in.  
```

Comment:
The empty line above needs to be removed, too for markdown to enumerate correctly.

```suggestion
2. A `u` tag with the value set to the normalized URL of the webpage they're commenting in.  
```